euphonic/cli/utils.py

```diff
@@ -439,12 +438,8 @@ def _brille_calc_modes_kwargs(args: Namespace) -> Dict[str, Any]:
     if args.n_threads is None:
         # Nothing specified, allow defaults
         return dict()
-    else:
-        if args.n_threads > 1:
-            parallel = True
-        else:
-            parallel = False
-        return dict(useparallel=parallel, threads=args.n_threads)
+
+    return dict(useparallel=(args.n_threads > 1), threads=args.n_threads)
```

Comment:
Might just be me, but I prefer the implicit rather than explicit dict.
```suggestion
    return {"useparallel": args.n_threads > 1, "threads": args.n_threads}
```

Reply:
In cases such as this, where the data is intended as kwargs, it is nice how the explicit `dict` more closely resembles the **kwargs in a function call.
When there are a lot of elements, it can also be noticeably more compact and easier to read (especially with syntax highlightling.)

But with two components, one of which is an expression, this is very marginal. Happy to accept suggestion.

In this particular case there are only